src/style.css

```diff
@@ -158,6 +158,22 @@ body {
   text-align: right;
 }
 
+/* comment pop up styles */
+.comment-popup {
+  margin-top: 100px;
+  min-height: 500px;
+  background-color: blue;
+  width: 100%;
```

Comment:
Here you could consider :bulb: adding width and height based on the viewport so the pop-up stays on top of everything, and fills the entire screen :100: